php/libraries/Candidate.class.inc

```diff
@@ -374,7 +406,12 @@ class Candidate implements \LORIS\StudyEntities\AccessibleResource,
      */
     function getListOfTimePoints(): array
     {
-        return $this->listOfTimePoints ?? [];
+        return array_map(
+            function ($timepoint) {
+                return intval($timepoint->getSessionID());
+            },
+            $this->data->getTimePoints(),
```

Comment:
I think we removed this function

Reply:
yeah phan caught it too now that tests are running again.